pkg/reconciler/trigger/trigger_test.go

```diff
@@ -141,6 +143,31 @@ func TestAllCases(t *testing.T) {
 					reconciletesting.WithTriggerBrokerFailed("DoesNotExist", "Broker does not exist"),
 				),
 			}},
+		}, {
+			Name: "Default broker not found, with injection annotation enabled",
```

Comment:
Should we also add a test for `Default broker found, with injection annotation enabled`?